src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +133,29 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_sections="dk_bench",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_sections="dk_bench",
+)
+@click.option(
+    "--system-prompt",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+)
+@click.option(
+    "--temperature",
```

Comment:
all of these flags are going to become hard to understand as we add more benchmarks. As a new user I would scan for `--dk-...` if I was trying to run a benchmark for the first time.